### DISCUSSION TITLE
Github Two-Factor Authentication Failed

### DISCUSSION THREAD

**Discussion by iqra-smith** [community/community#131221] [Other Feature Feedback, Questions, & Ideas]
Can you please help me get my account recovered? I entered the code from the Microsoft Authenticator App. But, I got an error. It said two-factor authentication failed. Please tell me why this occurred and how I can resolve this issue.

![screenshot-github com-2024 07 06-13_44_44](https://github.com/community/community/assets/171790305/fa5b653d-464b-4ae4-9ba4-65ebd3f3bc1e)

**Comment by davevad93** [comment]
Hi @iqra-smith ,

If you can't access you account with 2FA you should use your `Recovery codes`, if you've lost or haven't downloaded the codes you should contact support by opening a ticket [here](https://support.github.com/contact/account), select `Device verification on sign-in, unable to access email` and explain your issue.
